docs/guides/dataset/set_page_rank.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 3
+slug: /set_page_rank
+---
+
+# Set page rank
+
+Create a step-retrieval strategy using page rank.
+
+---
+
+## Scenario
+
+In an AI-powered chat, you can configure a chat assistant or an agent to respond using knowledge retrieved from multiple specified knowledge bases (datasets), provided that they employ the same embedding model. In situations where you prefer information from certain knowledge base(s) to take precedence or to be retrieved first, you can use RAGFlow's page rank feature to increase the ranking of chunks from these knowledge bases. For example, if you have configured a chat assistant to draw from two knowledge bases, knowledge base A for 2024 news and knowledge base B for 2023 news, but wish to prioritize news from year 2024, this feature is particularly useful.
+
+:::info NOTE
+It is important to note that this 'page rank' feature operates at the level of the entire knowledge base rather than on individual files or documents.
+:::
+
+## Configuration
+
+On the **Configuration** page of your knowledge base, drag the slider under **Page rank** to set the page rank value for your knowledge base. You are also allowed to input the intended page rank value in the field next to the slider.
+
+:::info NOTE
+The page rank value must be an integer. Range: [0,100]
+
+- 0: Disabled (Default)
+- A specific value: enabled
+:::
+
+:::tip NOTE
+If you set the page rank value to 1.7, it will be rounded down to 1.
```

Comment:
If you set the page rank value to non-integer, it will be rounded down to an integer.